Nvpmodel: bind the /usr/sbin/nvpmodel file for the jetson_clocks script to print the corresponding Power Mode

EstebanVg15 opened the pull request:
When running `jetson-clocks-show` the power mode should be printed after all the system-clock frequencies, but it was not being displayed since the `jetson_clocks` script did not find `/usr/sbin/nvpmodel`, so there is needed a layout to bind-mount this file so it can be accessible by the script.

EstebanVg15 committed:
Nvpmodel: bind the /usr/sbin/nvpmodel file for the jetson_clocks script to print the corresponding Power Mode

alexclewontin commented on nvpmodel/snap/snapcraft.yaml:
```diff
@@ -67,6 +67,8 @@ layout:
   /var/lib/nvpmodel:
     bind: $SNAP_DATA/var/lib/nvpmodel
+  /usr/sbin/nvpmodel:
+    bind-file: $SNAP/usr/sbin/nvpmodel
```
Is it critical that this is a bind-file? This is super nitpicky, but a `symlink:` layout is a bit more performant, if it works.

EstebanVg15 replied:
Hi @alexclewontin ,

Thank you for your feedback, I went directly through th `bind-file` layout, but I agree it is better to use `symlink`, I implemented and tested such change and it worked properly.

EstebanVg15 committed:
Nvpmodel: Use symlink instead of bind-file for the nvpmodel binary

 create a symbolic link is preferred because it is cheaper and more performant;
 bind-file could significantly increase the startup time of the app.

rmartin013 reviewed (approved):
Thanks for the change! LGTM
